src/mono/mono/metadata/metadata.c

```diff
@@ -1003,6 +1003,21 @@ mono_metadata_table_bounds_check_slow (MonoImage *image, int table_index, int to
         return mono_metadata_update_table_bounds_check (image, table_index, token_index);
 }
 
+static void
+compute_column_offsets (guint32 bitfield, guint8 column_offsets[MONO_TABLE_INFO_MAX_COLUMNS])
```

Comment:
nit: we don't usually use fixed length array types in function arguments in mono. we usually just use a pointer type. probably doesn't matter.  just stands out a bit.